recipes/cpp-taskflow/all/conanfile.py

```diff
@@ -22,30 +21,38 @@ class CppTaskflowConan(ConanFile):
     def configure(self):
         compiler = str(self.settings.compiler)
         compiler_version = tools.Version(self.settings.compiler.version)
+        min_req_cppstd = "17" if self.version <= "2.2.0" else "14"
```

Comment:
```suggestion
        min_req_cppstd = "17" if tools.Version(self.version) <= "2.2.0" else "14"
```

Need to convert before comparing